src/main/scala/introcourse/level07/LogParser.scala

```diff
@@ -105,7 +135,11 @@ object LogParser {
     * scala> getErrorsOverSeverity(List(KnownLog(Error(2), 123, some error msg")), 2)
     * > List()
     **/
-  def getErrorsOverSeverity(logs: List[LogMessage], minimumSeverity: Int): List[LogMessage] = ???
+  def getErrorsOverSeverity(logs: List[LogMessage], minimumSeverity: Int): List[LogMessage] = logs
+    .filter(_.isInstanceOf[KnownLog])
```

Comment:
TODO: Find out if filtering on instance type and casting is acceptable instead of pattern matching.
- If not, will likely have to refactor to use `.collect`